kytos/core/config.py

```diff
@@ -125,18 +131,60 @@ def parse_args(self):
         options, argv = self.conf_parser.parse_known_args()
 
         config = ConfigParser()
-        result = config.read([options.conf or defaults.get('conf')])
 
-        if result:
-            defaults.update(dict(config.items("daemon")))
-        else:
-            print('There is no config file.')
-            exit(-1)
+        config_file = options.conf or defaults.get('conf')
+
+        if not os.path.exists(config_file):
+            self._generate_file_from_template(TEMPLATE_FILES, BASE_ENV,
+                                              prefix=BASE_ENV,
+                                              syslog_args=SYSLOG_ARGS)
+
+        config.read(config_file)
+
+        defaults.update(dict(config.items("daemon")))
 
         self.parser.set_defaults(**defaults)
 
         self.options['daemon'] = self._parse_options(argv)
 
+    @staticmethod
+    def _generate_file_from_template(templates,
```

Comment:
`_generate_file_from_template` can be moved to a function instead of an `staticmethod`